CHANGELOG.md

```diff
@@ -1,5 +1,101 @@
 # Change Log
 
+## [0.9.0](https://github.com/confluentinc/ksql/releases/tag/v0.9.0-ksqldb) (2020-05-08)
+
+### Features
+
+* add multi-join expression support ([#5081](https://github.com/confluentinc/ksql/pull/5081)) ([002cd5a](https://github.com/confluentinc/ksql/commit/002cd5ad6473cbb000291e32ebe5a459d2870b61))
+* support more advanced suite of LIKE expressions ([#5013](https://github.com/confluentinc/ksql/pull/5013)) ([67cd9d9](https://github.com/confluentinc/ksql/commit/67cd9d91690039cc01f43438db1fbcd3fe4924ff))
+* add COALESCE function ([#4829](https://github.com/confluentinc/ksql/pull/4829)) ([251c237](https://github.com/confluentinc/ksql/commit/251c237c455f4cf3843628f41e4bbf57795cfd12))
+* add internal request to KsqlRequestConfig and SessionProperties ([#4771](https://github.com/confluentinc/ksql/pull/4771)) ([fc10cae](https://github.com/confluentinc/ksql/commit/fc10caeb6c164febd8041bc523e765fa6a6b77d8))
+* add KsqlQueryStatus to decouple from KafkaStreams.State ([#5029](https://github.com/confluentinc/ksql/pull/5029)) ([e8cbcde](https://github.com/confluentinc/ksql/commit/e8cbcde548e1f3078b1396173a3f55f86ee20626))
+* support multi-way joins ([#5064](https://github.com/confluentinc/ksql/pull/5064)) ([64dd39e](https://github.com/confluentinc/ksql/commit/64dd39ee77928221f08371d5e4424839d20c09f6))
+* Adds rate limiting to pull queries ([#4951](https://github.com/confluentinc/ksql/pull/4951)) ([6284111](https://github.com/confluentinc/ksql/commit/6284111652252b63cfc233bdfc3e08dbff983bbd))
+* CORS support for Vert.x based ksqlDB ([#4944](https://github.com/confluentinc/ksql/pull/4944)) ([5632893](https://github.com/confluentinc/ksql/commit/56328931d7eb26699fb5bd0940cfd03f7f9e53f9))
+* Do not allow access to new streaming endpoints using HTTP1.x ([#5193](https://github.com/confluentinc/ksql/pull/5193)) ([8b90035](https://github.com/confluentinc/ksql/commit/8b90035c6facf47d81dfd1616784b191141dce31))
```

Comment:
This should either be moved to the bug fix section or dropped entirely.

Reply:
Done